README.md

```diff
@@ -44,6 +44,7 @@ List of content:
 * [MinaExplorer API](https://graphql.minaexplorer.com/) Open GraphQL API from MinaExplorer
 * [Validators List](https://mina.staketab.com/) Overview of validators for mainnet
 * [MinaGraph API](https://minagraph.com) An alternative public GraphQL API node for Mina, exposed from the original GraphQL API of the Mina daemon
+* [Block Producer Rewards Calculator](https://towerstake.com/block-producer-rewards-calculator/) Calculate an expected return from running a validator node on Mina Protocol
```

Comment:
In general, we don't directly to the user, we add a description of the tool. Making an exception would sound odd imo. 

I would suggest changing this to: `Calculator for estimating the returns as a validator`

Reply:
Hi,
Thanks for the advice, i change the description.